src/types/UTF8OutPipeReader.cpp

```diff
@@ -30,24 +32,24 @@ UTF8OutPipeReader::UTF8OutPipeReader(HANDLE outPipe) :
     bool fSuccess{};
 
     // in case of early escaping
-    *_buffer = 0;
-    strView = std::string_view{ reinterpret_cast<char*>(_buffer), 0 };
+    _buffer.at(0) = 0;
+    strView = std::string_view{ reinterpret_cast<char*>(_buffer.at(0)), 0 };

```

Comment:
I'm afraid that'll just cast char '\0' to nullptr.
`strView = std::string_view{ reinterpret_cast<char*>(_buffer.data()), 0 };`

Reply:
Oh. Good call. I'll walk through and find instances of this and correct.